### PR TITLE
feature/CPRI-3992 PlacementFormValidator created and default value se…

### PR DESCRIPTION
**Pull request by Gia1987**
…t to Non-Prominent on Placement section

### CODE REVIEW

File: src/api/FieldValidator/validators/PlacementFormValidator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Validator from './Validator'
+
+class PlacementValidator extends Validator {
+  setLabel(value) {
+    this.label = value || 'Placement'
+  }
+  setValue(value) {
+    this.value = (value === undefined || value === '') ? 'Non-Prominent' : value
```
Comment (devu) [inline src/api/FieldValidator/validators/PlacementFormValidator.js]: Ideally this default option should come from taxonomies, not being hardcoded.

File: src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai'
+import { taxonomies } from 'model/taxonomies'
+import { PlacementFormValidator } from '..'
+import FieldMap from '../../FieldMap'
+
+const setup = (value) => {
+  const map = new FieldMap()
+
+  const validator = new PlacementFormValidator()
+  validator.setValue(value)
```
Comment (devu) [inline src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js]: can you please pass taxonomies here too so we can use it to set default values?

File: src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  validator.validate(map, taxonomies)
+
+  return {
+    validator
+  }
+}
+
+describe('PlacementFormValidator', () => {
```
Comment (sandeepnij90) [inline src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js]: we should add test coverage when value is empty or undefined

File: src/api/FieldValidator/validators/PlacementFormValidator.js (hunk 1 above)
Comment (Gia1987) [reply src/api/FieldValidator/validators/PlacementFormValidator.js]: done

File: src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js (hunk 2 above)
Comment (Gia1987) [reply src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js]: done

File: src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js (hunk 3 above)
Comment (Gia1987) [reply src/api/FieldValidator/validators/__test__/PlacementFormValidator.test.js]: The value will be by default 'Non-Prominent'. So, it could never be undefined or invalid value.
